src/ansys/mechanical/core/run.py

```diff
@@ -177,6 +177,13 @@ def cli(
             stacklevel=2,
         )
 
+        if version < 241:
+            warnings.warn(
```

Comment:
@klmcadams looks like the warning is duplicated, and I  think warning only makes sense in batch mode

Reply:
Ohh okay, I thought we wanted it with -script. I'll change it for batch mode only

Reply:
@klmcadams yep, its for the combination of -i and -b